torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -1531,13 +1531,12 @@ Tensor* TensorExprKernel::computeValue(const torch::jit::Value* v) {
   }
 }
 
-Stmt* TensorExprKernel::generateStmt(BackendType backendType) {
-  // Find all tensors we need to compute (including dependencies) and put them
-  // in a topological order
-  std::vector<Tensor*> tensors_to_compute =
-      findAllNeededTensors(tensorOutputs_);
-
-  torch::jit::tensorexpr::LoopNest l(tensorOutputs_, tensors_to_compute);
+Stmt* TensorExprKernel::generateStmt(BackendType backendType, Stmt* st) {
```

Comment:
Nit: previously this method generated a statement and applied loop transformations to it, now it only applies loop transformations. We probably should rename it to reflect this.

Reply:
Good point. Renamed.